style.css

```
@@ -0,0 +1,177 @@
+:root {
+  --color-Lav: #EEE2FE;
+  --color-hov: #4D62CD
+  
+}
+
+header {
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-right: 30px;
+  background-image: url(https://cdn.glitch.com/4a049c3c-7701-4771-9366-0d6fff4d027c%2FBlobs_Title.svg?v=1607395375106);
+  background-size: 500px; 
+  background-position: 35% 90%;
+  background-repeat: no-repeat;
+  
+}
+
+body {
+  font-family: 'IBM Plex Sans', sans-serif;
+  background-color: var(--color-Lav);
+  max-width: 100ch;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.clearfix::before,
+.clearfix::after {
+  content: " ";
+  display: table;
+}
+.clearfix::after {
+  clear: both;
+}
+
+h1 {
+  font-size: 45px;
+}
+
+h2{
+  font-size: 32px;
+}
+
+h3{
+  font-size: 25px;
+  text-decoration: underline wavy;
+  text-decoration-skip: none;
```

You want the property `text-decoration-skip-ink` here:
```suggestion
  text-decoration-skip-ink: none;
```